### PR TITLE
Global params revamp; #684

### PR DESCRIPTION
**Pull request by alrusdi**
YA attempt to make compact global parameters
More translations for colonies
Highlight hint for SelectSpace action
Снимок экрана от 2020-06-04 22-15-32
Снимок экрана от 2020-06-04 22-16-35

### CODE REVIEW

File: src/components/BoardMixin.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const BoardMixin = {
             return boardSpaces;
         },
         getMainSpaces: function (): Array<SpaceModel> {
-            return (this as any).getAllSpaces().filter((s: SpaceModel) => {return parseInt(s.id) < 70})
+            return (this as any).getAllSpaces().filter((s: SpaceModel) => {return s.spaceType != "colony"})
```
Comment (bafolts) [inline src/components/BoardMixin.ts:21]: We should use `SpaceType.COLONY` instead of this string.

**Comment by funkybaboon** [comment]
@alrusdi The free ocean at 0°C is missing on the new map.

**Comment by alrusdi** [comment]
@funkybaboon yes, i've fixed it already. Waiting for PR approovement

**Comment by ssimeonoff** [comment]
@alrusdi , молодец!

Best UI for the global params. (first time I saw it [here](https://scontent-arn2-1.xx.fbcdn.net/v/t1.15752-9/66753914_315487199169615_5355324404177829888_n.jpg?_nc_cat=104&_nc_sid=b96e70&_nc_ohc=bQjAccJyLbEAX-asq1n&_nc_ht=scontent-arn2-1.xx&oh=7a260546ca75048632ea8451c8e550c6&oe=5F1ABC4C) )

**Comment by nwai90** [comment]
@alrusdi Suggestion: Why not use a different colour instead of applying opacity? Attached an example below, I feel it's quite clear that oxygen is at 0%, Venus is at 2%, and temperature is at -26 degrees.

Screenshot 2020-06-25 at 8 19 03 PM

**Comment by alrusdi** [comment]
@nwai90 but problem is how to distinquish a **-26** and **-24** values, not how to read exact value. Prev variant was more readable.

**Comment by nwai90** [comment]
@alrusdi I see. Was thinking in relation to https://github.com/bafolts/terraforming-mars/issues/777, that full opacity might be helpful in this direction.

**Comment by ssimeonoff** [comment]
Maybe add colour and text-shadow to the active ones, something like: 
Untitled

**Comment by alrusdi** [comment]
@nwai90 look at it! It can be great addition to the frame around current value

**Comment by nwai90** [comment]
@ssimeonoff Thanks for illustration! I'll update and post

**Comment by ssimeonoff** [comment]
wouldn't it be better to keep the inactive ones with the opacity 0.4 as they were?

**Comment by nwai90** [comment]
> wouldn't it be better to keep the inactive ones with the opacity 0.4 as they were?

It's subjective; 0.4 makes it quite hard to see IMO. If not full opacity, perhaps a higher value closer to 1 would improve readability (e.g. 0.8)

**Pull request merged**